apis/config/v1beta1/configuration_types.go

```diff
@@ -169,6 +169,17 @@ type ControllerMetrics struct {
 	// metrics will be reported.
 	// +optional
 	EnableClusterQueueResources bool `json:"enableClusterQueueResources,omitempty"`
+
+	// +optional
+	LocalQueueMetrics *LocalQueueMetrics `json:"localQueueMetrics,omitempty"`
+}
+
+type LocalQueueMetrics struct {
+	// +optional
+	Enable bool `json:"enable,omitempty"`
+
+	// +optional
+	LocalQueueSelector *metav1.LabelSelector `json:"localQueueSelector,omitempty"`
```

Comment:
Please also clarify what is the default. I think it would be safer to diable the LQ metrics by default to be on the safe side, either by the selector or the enable field. However, it would be good to clarify in the comments.